tests/response-metadata.js

```diff
@@ -38,7 +38,7 @@ describe("Promise", function () {
         var response = await imagekit.getPurgeCacheStatus(requestId);
         expect(response).to.be.deep.equal(dummyAPISuccessResponse);
         expect(response.$ResponseMetadata.statusCode).to.be.equal(200);
-        expect(response.$ResponseMetadata.headers).to.be.deep.equal({
+        expect(Object.fromEntries(response.$ResponseMetadata.headers)).to.be.deep.equal({
```

Comment:
Why is this needed? Seems like a breaking change.

Reply:
it was needed because we upgraded `axios`  to `v1.6.5` and axios was sending `response header` differently but reverted the `axios` upgrade for now

Reply:
why was axios upgraded? And if we had to upgrade axios for some reason, we could have done `Object.fromEntries` inside the SDK where we set `response.$ResponseMetadata.headers` to avoid any breaking change.